homeassistant/data_entry_flow.py

```diff
@@ -454,14 +454,16 @@ def show_advanced_options(self) -> bool:
         return self.context.get("show_advanced_options", False)
 
     def add_suggested_values_to_schema(
-        self, data_schema: vol.Schema, suggested_values: Mapping[str, Any]
+        self, data_schema: vol.Schema, suggested_values: Mapping[str, Any] | None
```

Comment:
I think it makes sense to add `None` support here, as user_input in config flows is usually optional.
It removes the need for a check in every integration.

Reply:
Moved to #85763 to avoid core changes inside the PR.